scripts/cluster_network.py

```diff
@@ -605,12 +580,20 @@ def clustering_for_n_clusters(
     clustering = get_clustering_from_busmap(
         n,
         busmap,
-        bus_strategies=bus_strategies,
         aggregate_generators_weighted=True,
         aggregate_generators_carriers=aggregate_carriers,
         aggregate_one_ports=["Load", "StorageUnit"],
         line_length_factor=line_length_factor,
+        line_strategies=line_strategies,
         generator_strategies=generator_strategies,
+        bus_strategies={
+            "lat": "mean",
+            "lon": "mean",
+            "tag_substation": "first",
+            "tag_area": "first",
+            "country": "first",
+        },
```

Comment:
This is a bit widely duplicated across scripts. v_nom is also not here, may this be somewhat linked to the issue by Emmanuel?

Reply:
Agree that the duplication should be fixed. Implemented in #1065 as [an update](https://github.com/ekatef/pypsa-earth/blob/729fecf24760dff828a1e7ee159885432108a79d/scripts/cluster_network.py#L745-L756) of the aggregation strategies outside the clustering functions.

Yeah, it looks like transparence is much desired in working with `v_nom`. Have defined `"first"` aggregation strategy for `v_nom` to have a better overview on what is put inside the model.

Reply:
> Good! What would be your idea here, merge this and then 1065, merge 1065 here?

I think it could make sense to have a stacked PR: [a PR on top](https://github.com/ekatef/pypsa-earth/pull/1) of #1065 which enables linopy in addition to the upgrade of PyPSA version